dev-vcs/tig/tig-2.2.1.recipe

```diff
@@ -46,11 +55,23 @@ BUILD()
 {
 	libtoolize --force --copy --install
 	./autogen.sh
-	runConfigure ./configure
+	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
 	make
 }
 
 INSTALL()
 {
 	make install
+
+	# temporary solution because of xmlto deps failing
```

Comment:
We have xmlto, so maybe it would be better to try and use it, to avoid the manual installation of the man pages.
I'm not sure, but I _think_ I managed to call "make install install-doc-man" a few days ago (after having added a **`cmd:xmlto`** to BUILD_PREREQUIRES).
(I might dig this issue if I have time.)